ui-grid/ui-grid.d.ts

```diff
@@ -1064,6 +1074,15 @@ declare module uiGrid {
         (scrollEvent: JQueryMouseEventObject): void;
     }
 
+    export interface sortChangedHandler {
+        /**
+         * Sort change event callback
+         * @param {IGridInstance} grid instance
+         * @param {IGridColumn} array of gridColumns that have sorting on them, sorted in priority order
+         */
+        (grid: IGridInstance, columns: IGridColumn[]): void;
```

Comment:
You can just edit the code in your branch in your own repository and push the changes to your repository. It will automatically be picked up and included in this PR. That's how it normally works, at least. I see it says "unknown repository", so I'm a bit unsure if it works in your case. Is your repository private? It's worth a try, anyway. :)